api/admin/controller.py

```diff
@@ -228,6 +230,33 @@ def edit(self, data_source, identifier):
             )
             changed = True
 
+        new_target_age_min = flask.request.form.get("target_age_min")
+        new_target_age_max = flask.request.form.get("target_age_max")
+        if new_target_age_max < new_target_age_min:
+            return INVALID_EDIT.detailed("Minimum target age must be less than maximum target age.")
+
+        if work.target_age:
+            old_target_age_min = work.target_age.lower
+            old_target_age_max = work.target_age.upper
+        else:
+            old_target_age_min = None
+            old_target_age_max = None
+        if new_target_age_min != old_target_age_min or new_target_age_max != old_target_age_max:
+            # Delete all previous staff target age classifications
+            for c in old_classifications:
+                if c.subject.type == Subject.AGE_RANGE:
+                    self._db.delete(c)
+
+            # Create a new classification with a high weight - higher than audience
+            age_range_identifier = "%s-%s" % (new_target_age_min, new_target_age_max)
+            work.primary_edition.primary_identifier.classify(
+                data_source=staff_data_source,
+                subject_type=Subject.AGE_RANGE,
+                subject_identifier=age_range_identifier,
+                weight=10000000,
```

Comment:
Maybe make this a constant since you'll be using this weight for genres and audience as well?